core/docs/api/options.md

```diff
@@ -17,7 +17,7 @@ value.
 
 -------------------------------------------------------------------------------
 
-# km_core_option_scope enum {#km_core_option_scope}
+# km_core_option_scope enum
```

Comment:
why was the # anchor removed?

Reply:
OK, I can see from https://help.keyman.com/developer/core/current-version/options#toc-km-core-option-scope-enum-km-core-option-scope- that this was broken before. It's an extension that's supposed to fix the anchor's identity. So this is a good change. But it should be noted in the PR description.

Reply:
Noted. I will write it in the PR description.